### ISSUE TITLE
If 1 thread per account, scan speed should be faster

### ISSUE THREAD

**Issue by DomGrieco** [AHAAAAAAA/PokemonGo-Map#3111]
I've been doing a lot of testing today with the dev branch and I'm starting to notice that it doesn't scan as fast as it should.

## Expected Behavior
1 thread per account
400 accounts
st 50 = 7351 steps
Full scan should take a minimum of 94.91 seconds at best, but that isn't realistic. 

## Current Behavior
1 thread per account
400 accounts
st 50 = 7351 steps
Full scan takes anywhere from 7 to 9 minutes. 

## Possible Solution
Don't know yet, will update if I find one.

## Your Environment

* Version used: 4dba6e6d86fbdbcd4116896732f5c635db1fd691
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Ubuntu 16.04 x64

**Comment by ELusiveLPFAN** [comment]
The problem is that not all accounts successfully login's to server, 10-30% will fail and account's will banned for 15min, but if workers continue running, they will attemt to login again, and 15min stay for while.

**Comment by Dagothur** [comment]
@ELusiveLPFAN what triggers accounts being banned? 

is there a way to detect that an account is banned??

**Comment by ELusiveLPFAN** [comment]
@Dagothur, they will have no access for 15 minutes, just stop scanning and wait. Use logs for watching about errors/bans.

**Comment by ELusiveLPFAN** [comment]
Yes, and after I can't login at Pokemon club, it say that I tried login many times and failed, so they disable account for 15 minutes.

**Comment by ELusiveLPFAN** [comment]
After 15 minutes I successfully login from this account.

**Comment by Dagothur** [comment]
Okay, thanks. Looks like some of my workers were suffering from a different issue.

**Comment by excitedbox** [comment]
The only time I got that message I had made a typo in the username. I do get a lot of failed logins and retries though. Sometimes 10+ for some accounts. On one scan logging in 100 accounts took over 15 minutes.

**Comment by tainaan** [comment]
Might be the wrong place to ask but how the hell do you guys get so many accounts? It takes so long to individually set up a google account plus it starts requiring phone numbers after a certain point.
